authors/apps/articles/serializers.py

```diff
@@ -84,6 +84,7 @@ class ArticlePaginator(PageNumberPagination):
 
     def get_paginated_response(self, data):
         return Response(OrderedDict([
+            ("pageArticlesCount", len(self.page.object_list)),
```

Comment:
pageArticlesCount is ambiguous. why not just pageCount or articlesPageCount?

Reply:
Let me fix that @oluwagbenga-joloko